custom_components/eldes_alarm/__init__.py

```diff
@@ -1,14 +1,13 @@
 """Support for the Eldes API."""
-from datetime import timedelta
-import logging
 import asyncio
+import logging
+from datetime import timedelta
```

Comment:
Try not to change places you don't need to change. Keep imports in same places, this makes your PR cleaner.

Reply:
Sorry I turned on jetbrains import optimizer, will remove all those reorderings